dpx/src/specials/html.rs

```diff
@@ -737,76 +720,60 @@ unsafe fn spc_handler_html_default(mut spe: *mut spc_env, mut ap: *mut spc_arg)
     error
 }
 /* translate wsp* '(' wsp* number (comma-wsp number)? wsp* ')' */
```

Comment:
Another piece of code where `nom` might make things a lot cleaner. The parse traits would be usable, with some small wrappers that clone the slice pointer, mutate it, subtract the length from the input and return the remainder and output as a `nom::IResult` by converting to a basic nom error. Then functions like these would be a few lines long, using nom combinators (except the end bit that mutates, just return the data and use it to mutate elsewhere).